### ISSUE TITLE
wnen I run python onnx2trt.py，it occurs a mistake

### ISSUE THREAD

**Issue by xuhui1994** [k9ele7en/ONNX-TensorRT-Inference-CRAFT-pytorch#1]
![image](https://user-images.githubusercontent.com/35833743/146500469-ca385ed1-d17d-4304-b132-1216d829b0a8.png)

**Comment by xuhui1994** [comment]
> ![image](https://user-images.githubusercontent.com/35833743/146500469-ca385ed1-d17d-4304-b132-1216d829b0a8.png)

sorry, it's nothing with onnx_trt.py, It occurs a mistake above when I run python infer_trt.py.

**Comment by k9ele7en** [comment]
Hi @xuhui1994 , I have noted that infer code using TRT are currently return wrong output. I don't have time to debug it right away, but will do it soon. Thanks.
<img width="829" alt="image" src="https://user-images.githubusercontent.com/53246028/146501104-0b315e20-3d15-44e3-9033-a71776ba43e3.png">

**Comment by xuhui1994** [comment]
> Hi @xuhui1994 , I have noted that infer code using TRT are currently return wrong output. I don't have time to debug it right away, but will do it soon. Thanks. <img alt="image" width="829" src="https://user-images.githubusercontent.com/53246028/146501104-0b315e20-3d15-44e3-9033-a71776ba43e3.png">

yes, what you say above is right. How can I get a correct ouput when run infer_trt.py ?Can you have a good solution?

**Comment by k9ele7en** [comment]
Yes, I have worked on it a lot, a problem is in the inference step on GPU (load RT engine, host<->device data moving...). I know that because when I using the same RT engine converted from this to put in Triton Server's Model Repository, it works well.
It would be nice if you can have a look at Tensor RT inference example and debug the code. If you have installed TensorRT, example are in path: /usr/src/tensorrt/samples/. Also checkout this:https://docs.nvidia.com/deeplearning/tensorrt/sample-support-guide/index.html#mnist_sample. If the code run well, you can post a PR so that people can find useful later. Thanks @xuhui1994

**Comment by k9ele7en** [comment]
@xuhui1994 TensorRT also update new lib called Torch-TensorRT that flexible build and load engine right in python code, you can check this out, I'm gonna update the repo with this method soon... https://developer.nvidia.com/blog/accelerating-inference-up-to-6x-faster-in-pytorch-with-torch-tensorrt/

**Comment by xuhui1994** [comment]
Ok , I try my best to make it. Thank you for reply.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "k9ele7en/ONNX-TensorRT-Inference-CRAFT-pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月17日(星期五) 下午2:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [k9ele7en/ONNX-TensorRT-Inference-CRAFT-pytorch] wnen I run python onnx2trt.py，it occurs a mistake (Issue #1)





 
Yes, I have worked on it a lot, a problem is in the inference step on GPU (load RT engine, host<-&gt;device data moving...). I know that because when I using the same RT engine converted from this to put in Triton Server's Model Repository, it works well.
 It would be nice if you can have a look at Tensor RT inference example and debug the code. If you have installed TensorRT, example are in path: /usr/src/tensorrt/samples/. If the code run well, you can post a PR so that people can find useful later. Thanks @xuhui1994
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
